Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:latest
```

Comment:
No, the `-v ./tls:/tls` effectively creates a mount point for the directory whether or it exists.